eng/imports/HostAgnostic.props

```diff
@@ -36,12 +36,16 @@
 
     <!-- CPS -->
     <PackageReference Include="Microsoft.VisualStudio.ProjectSystem" />
+    <PackageReference Include="Microsoft.VisualStudio.ProjectSystem.Implementation" />
```

Comment:
The implementation package should only be used by unit tests projects, not by product projects. The risk is to accidentally take dependencies on Implementation code.